OlsztynHooligans/src/app/bookmarks/bookmarks.component.ts

```diff
@@ -99,9 +127,105 @@ export class BookmarksPopupComponent implements OnInit {
     console.log(this.bookmarkForm);
     this.dataService.postbookmark(this.bookmarkModel).subscribe(res => {
               this.bookmarkModel = { text: this.bookmarkModel.text, title: this.bookmarkModel.title };
-              this.dialogRef.close();
+              this.dialogRef.close(res);
     }, (err) => {
       console.log(err);
     });
   }
 }
+
+@Component({
```

Comment:
plik powinien byc per component a nie 2 w jednym pliku

Reply:
korzystaliśmy z angular material i w dokumentacji w przykładzie jak zrobić popup było że w jednym pliku jest więcej komponentów https://stackblitz.com/angular/njdkroaomnb?file=app%2Fdialog-overview-example.ts

Reply:
bo to przykład aby nie robić więcej plików :) Powinno być oddzielnie :)